### PR TITLE
Nullsafety and merge chewie 1.0.0

### LINKED ISSUE #3: Not support MP3 in iOS ?
When I play a MP3 file in iOS, the log show the bug "The getter '_duration' was called on null". It seem that the plugin use something only support video in iOS. By the way, android is worked well.

### PR DESCRIPTION
**Pull request by tneotia**
I let the migration tool decide everything for this one - I tried to force some changes with `late` to reduce the number of `!` but they didn't work. 

Fixes #3 and possibly #1? Don't know for sure.

### CODE REVIEW

**Comment by erickok** [comment]
Looks decent on first glance. Indeed in some cases it _looks_ like force unwrapping could be reduced, but I'd have to look into detail in that.

**Comment by jhb-dev** [comment]
> Looks decent on first glance. Indeed in some cases it _looks_ like force unwrapping could be reduced, but I'd have to look into detail in that.

Hey @erickok 
Do you have time to look into that in the next days? Because we are waiting for this library to be migrated before flutter_html can be migrated 😄

**Comment by tneotia** [comment]
@erickok now I'm using chewies nullsafety migration in this PR and thus have merged the changes. I removed all the video related things so the experience should be very similar to before. The merge also fixes some issues noted in OP.

**Comment by erickok** [comment]
Thanks so much for this work @tneotia - it is a pretty big task. I have tested your changes (against Flutter 2.0.1) and it is generally working well. I am going to merge.

**Comment by erickok** [comment]
Actually I don't have the right to merge. @ryan-berger has to do this.

File: pubspec.yaml [hunk 1]
```diff
@@ -9,20 +9,19 @@ authors:
   - Eric Kok <erickok@gmail.com>
```
Comment (erickok) [inline pubspec.yaml:10]: Can you bump the version here to 1.2.0 as well? Then all we need for a release is to merge this PR.

Comment (tneotia) [reply pubspec.yaml:10]: Done. Should be good to go!

**Comment by erickok** [comment]
@ryan-berger could you please merge this? Then I will release.

**Comment by jlubeck** [comment]
Hey @erickok any estimates on when that release is going to happen? Thanks so much for all the work here guys!

**Comment by erickok** [comment]
It's release as 1.2.0.

Thanks again @tneotia - all credits to you. And thanks to @ryan-berger  for merging.

**Pull request merged**